packaging/os/rhn_register.py

```diff
@@ -119,7 +119,8 @@
 try:
     import up2date_client
     import up2date_client.config
-except ImportError, e:
+except ImportError:
+    e = get_exception()
```

Comment:
I think we may have to move the ansible.module_utils imports up above this otherwise get_exception won't be defined yet.  Could convert the ansible.module_utils imports to not be wildcard imports at the same time.